src/annotator/guest.js

```diff
@@ -106,16 +109,11 @@ export default class Guest extends Delegator {
    * @param {HTMLElement} element -
    *   The root element in which the `Guest` instance should be able to anchor
    *   or create annotations. In an ordinary web page this typically `document.body`.
-   * @param {Record<string, any>} config
-   * @param {typeof htmlAnchoring} anchoring - Anchoring implementation
+   * @param {Record<string, any>} [config]
```

Comment:
In future it will be possible to replace the generic `Record<string,any>` type with a more specific one which lists only the fields we actually use. This is for a follow-up PR though.